src/pages/style-guide.mdx

```diff
@@ -0,0 +1,480 @@
+import Condaforge1 from "@site/static/img/logo1.svg";
+import Condaforge2 from "@site/static/img/logo2.svg";
+import Condaforge3 from "@site/static/img/logo3.svg";
```

Comment:
Cool, will change the file names accordingly. I will have to capitalize the first letter for each file name because Docusaurus only imports SVG file names in mdx starting with capital letter. Something strange I noticed, but yeah.